packages/client/hmi-client/src/types/common.ts

```diff
@@ -106,9 +106,14 @@ export type SidePanelTab = {
 // Tabs
 export type Tab = {
 	icon?: string;
-	assetId?: string;
 	assetName?: string;
-	pageType?: AssetType | ProjectPages;
+	assetId: string;
+	pageType: AssetType | ProjectPages;
+};
```

Comment:
Why not extending `Tab` with `AssetRoute`?
Also why do we need `Tab`?

Reply:
I mentioned the use of `Tab` in the description - though I just renamed it to `AssetItem` to make things more clear. Let me know your thoughts.